src/components/utility/PlacePhoto.js

```diff
@@ -0,0 +1,8 @@
+export default function PlacePhoto(photo_reference) {
```

Comment:
if this is only used in the Map? in that case, you can put it in that folder. utility is more for stuff that's usable in other places

Reply:
I think we resolved this at today's meeting, he can use it on the home page too, specifically in the **`Image List`** I've been working on